### DISCUSSION TITLE
watermark logo

### DISCUSSION THREAD

**Discussion by TheFilthyFrog** [shiyiya/oplayer#87] [Q&A] [answered]
is there support for watermark overlay on the play?

**Comment by shiyiya** [comment, marked as answer]
@TheFilthyFrog 

```shell 
pnpm i @oplayer/ui@1.2.29-beta.0
```
https://github.com/shiyiya/oplayer/blob/cc0c140389fc634b757799cdde7f45c6cd94f95e/examples/standalone/main.ts#L58
